src/ui/public/directives/rows.js

```diff
@@ -40,12 +40,7 @@ module.directive('kbnRows', function ($compile, $rootScope, getAppState, Private
         let $cellContent;
 
         if (contents instanceof AggConfigResult) {
-          const isCellContentFilterable =
-            contents.type === 'bucket'
-            && contents.aggConfig.getField()
-            && contents.aggConfig.getField().filterable;
-
-          if (isCellContentFilterable) {
+          if (contents.type === 'bucket') {
```

Comment:
Do I really need to check if `contents.type === 'bucket'` if I'm checking `_.get(contents, 'aggConfig.type.createFilter')`?